apps/docs/src/app/core/component-docs/resizable-card-layout/examples/resizable-card-layout-example.component.scss

```diff
@@ -0,0 +1,8 @@
+.fd-docs-cursor {
+    cursor: auto !important;
+}
+
+.fd-docs-center-align {
+    display: flex;
+    justify-content: center;
+}
```

Comment:
this is repeated 3 times. You can make a single scss file and import it in the 3 files, no need to create a separate file for each component. Scss files can be reused

Reply:
using one scss file now. removed others.